src/user-guide/running-workflows/dynamic-behaviour.rst

```diff
@@ -3,19 +3,18 @@
 Cylc Broadcast
 --------------
 
-The ``cylc broadcast`` command overrides :cylc:conf:`[runtime]`
-settings in a running workflow.
-
-Broadcast can be used to communicate information to downstream tasks by
-setting environment variables. Broadcasts can target all tasks, groups of
-tasks, or specific tasks, at all cycle points or at specific cycle points.
-See broadcast command help for details.
-
-Broadcast settings targeted at a specific task ID or cycle point expire and
-are forgotten as the workflow moves on. Un-targeted variables and those
-targeted at a task name persist throughout the workflow run, even across
-restarts, unless manually cleared using the broadcast command - and so
-should be used sparingly.
+The ``cylc broadcast`` command overrides task :cylc:conf:`[runtime]`
+settings in a running scheduler. You can think of it as communicating
+changed settings (including information via environment variables) to selected
+upcoming tasks via the scheduler.
+
+Broadcasts can target all tasks, groups of tasks, or specific tasks, at all
+cycle points or at specific cycle points. See ``cylc broadcast --help`` for
+detailed information.
+
+Broadcast settings targeting a specific task ID or cycle point expire as the
```

Comment:
The task ID is irrelevant here isn't it (it may be a specific ID at all cycle points)?

Reply:
No, task ID includes cycle point.  So you can broadcast to a specific task instance.

Reply:
Ah, OK. Got confused because you refer to "specific tasks" in the paragraph above.
Still now sure we really need to mention task IDs here though.
If we just say "settings targeting a specific cycle point" that would include task IDs wouldn't it?

Reply:
Yes, fair enough. Especially as the broadcast CLI doesn't take task IDs as such (it takes cycle points and/or names, which can single out specific task IDs). I'll tweak the wording...

Reply:
Done.